models/pipeline.js

```diff
@@ -90,7 +90,19 @@ const MODEL = {
                 actions: Joi.array().items(Joi.string())
             })
         )
-        .description('List of subscribed scm urls paired with actions')
+        .description('List of subscribed scm urls paired with actions'),
+
+    sonarDashboardName: Joi.string()
```

Comment:
what if we have more integrations in the future ? Do we need a flat structure like this ? ie keep adding more fields for potential integrations like Saucelabs Project UI, Mend UI, Semgrep etc ?

Reply:
Updated